source/subs_functions/subs_functions.bas

```diff
@@ -1325,11 +1325,11 @@ clearid
 id.n = qb64prefix$ + "LoadFont": id.Dependency = DEPENDENCY_LOADFONT
 id.subfunc = 1
 id.callname = "func__loadfont"
-id.args = 3
-id.arg = MKL$(STRINGTYPE - ISPOINTER) + MKL$(DOUBLETYPE - ISPOINTER) + MKL$(STRINGTYPE - ISPOINTER)
-id.specialformat = "?,?[,?]"
+id.args = 4
+id.arg = MKL$(STRINGTYPE - ISPOINTER) + MKL$(LONGTYPE - ISPOINTER) + MKL$(STRINGTYPE - ISPOINTER) + MKL$(LONGTYPE - ISPOINTER)
+id.specialformat = "?,?[,[?][,[?]]]"
 id.ret = LONGTYPE - ISPOINTER
-id.hr_syntax = "_LOADFONT(fileName$, size%[, " + CHR$(34) + "{MONOSPACE|, BOLD|, ITALIC|, UNDERLINE|, UNICODE|, DONTBLEND}" + CHR$(34) + "])"
+id.hr_syntax = "_LOADFONT(fileName$, size&[, " + CHR$(34) + "{MONOSPACE|, UNICODE|, DONTBLEND|, MEMORY}" + CHR$(34) + "][, fontIndex])"

```

Comment:
I think this should match the `specialformat` in terms of where the brackets are. So, a pair of brackets around the string parameter, and the first `[,` should be closed out after the last parameter, not before it.

Reply:
Just curious: What happened to the Bold, Underline, Italic here?  From what I remember, they never worked anyway; have we just removed them completely?  How will that affect existing programs that someone might have that references them?  Even though I don't currently think they do anything, I think they still compile without tossing any sort of error messages.

Reply:
Well, they were not really doing anything. So, no point in advertising that stuff. We can still pass those, but it will be ignored just like the old code did. So, no changes in the way those were handled.

Reply:
Sounds good.   I just noticed the change and was kinda worried we'd error out with them now.  Ignoring them is the proper thing to do, IMHO, I think.  ;)

Reply:
I wrote those `hr_syntax` help texts in that way to keep thing consistent with how it was done before. Like this:
https://github.com/QB64-Phoenix-Edition/QB64pe/blob/17b8fe4ac4dcd64c4d292c2191b8d6ba3dfe2fd5/source/subs_functions/subs_functions.bas#L3067

I guess the previous authors wanted to keep the help text simple for users to understand. 🤷‍♂️